### PR TITLE
Slide-in animation with more information added

### LINKED ISSUE #6: Learn More should have an opaque background
The translucent background reduces readability:
screen shot 2016-07-28 at 11 12 24 am
Maybe if we blurred the background we wouldn't have this problem, but that seems like too much work for the benefit.

### PR DESCRIPTION
**Pull request by srustagi**
Added responsive meta tag to index.html and fixed css reset issues. This resolves #6. PTAL @gmittal

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 $("#more").click(function(){
     $("section").css({"position": "fixed"});
-    $("#info").animate({"left": "10%",}, 200);
-    $("#info").animate({"left": "0%",}, 200);
+    $("#info").animate({"left": "0%"}, 200);
```
Comment (ankitr) [inline js/main.js]: Is it possible to run these simultaneously? Otherwise it looks kinda weird.

Comment (srustagi) [inline js/main.js]: @ankitr that was the bounce effect. @gmittal suggested that I remove it to keep the animation serious. This isn't up to date?!

Comment (ankitr) [inline js/main.js]: Btw does anyone know the state of CSS animations? @gmittal

Comment (srustagi) [inline js/main.js]: these are JQuery, i kept it that way because not all browsers have CSS3 translate support :P

Comment (ankitr) [inline js/main.js]: Not talking about the bounce effect. I'm talking about the shift left and the opacity change. They don't happen simultaneously and end up looking stilted.

**Comment by ankitr** [comment]
When the site is initially loaded, it looks like the info section just fades in instead of moves in from the left.

EDIT: you're one step ahead of me!

File: js/main.js (hunk 1 above)
Comment (srustagi) [inline js/main.js]: Is it fixed now?

**Comment by gmittal** [comment]
I feel like the info content should be centered as well. It looks a little off right now.

**Comment by gmittal** [comment]
This is minor, but I think the Home and Learn More links should have the same font. Also, we don't want an icon for the home link (I would suggest using another `&lsaquo;`

**Comment by gmittal** [comment]
Maybe make the info body text grey? (adds a nice contrast between the title and the body)

**Comment by gmittal** [comment]
Add the [`pills.min.css`](https://cdn.rawgit.com/rohitkrai03/pills/gh-pages/dist/pills.min.css) file and set the body text div class to `eight columns`. That will make the text less spread out on larger screens. 

The title seems a little big to me. Maybe try and mimic the layout and font sizes of [this page](https://getbootstrap.com/examples/sticky-footer/).

**Comment by ankitr** [comment]
This will resolve #6.

File: js/main.js (hunk 1 above)
Comment (ankitr) [inline js/main.js]: Actually, let's make it synchronous. First the home fades out, then the info slides in. Might make sense to speed things up a bit to make it still seem quick. Thoughts?

**Comment by gmittal** [comment]
Status on this @srustagi?

**Comment by ankitr** [comment]
We're gonna need to rebase from hardhat to get this up to date. After that, LGTM. Wait for the go-ahead from @gmittal. Nice job @srustagi.

**Comment by ankitr** [comment]
Looks like you'll have to open a new PR to merge with hardhat and not master.

**Comment by gmittal** [comment]
The info text should be left-aligned (or justified). I'm still not super satisfied with the animation transition nor the home link, but I think it's fine for now. 

Congrats @srustagi, you have received your first LGTM (sans the text alignment thing which @ankitr should confirm) .

**Comment by ankitr** [comment]
You can push that to the next PR. Reopen this as a PR to merge with hardhat

**Comment by gmittal** [comment]
In that case, LGTM.

**Comment by gmittal** [comment]
@ankitr @srustagi this should really get merged sometime soon.

**Comment by ankitr** [comment]
@gmittal PTAL I reset everything to the base branch

File: css/index.css [hunk 2]
```diff
@@ -124,3 +123,8 @@ button {
     margin-top: 1rem;
 }
 
+=======
+#info h1{
+    font-size: 32px;
+}
+>>>>>>> fixed font color, size, padding, made animations happen together, and centered text. also optimized for mobile screens
```
Comment (gmittal) [inline css/index.css]: Remnants of the merge.

File: index.html [hunk 3]
```diff
@@ -16,8 +17,15 @@
...
-                    <p> Bresaola shoulder sausage, alcatra swine ground round prosciutto strip steak rump. Chuck meatball ham pancetta pork belly. Turducken brisket swine landjaeger pork short loin. Capicola jerky rump andouille ham. Pastrami cupim turducken pork jerky t-bone, turkey pork chop capicola kielbasa beef brisket shoulder ham hock cow. Pork chop landjaeger shankle pancetta, kevin flank prosciutto porchetta leberkas tenderloin corned beef jerky short ribs.</p>
+                    <p>
+                        Bacon ipsum dolor amet kielbasa chuck short loin, tenderloin sirloin beef ribs biltong jerky ground round leberkas pork chop. Hamburger ball tip turducken, chuck cupim bresaola beef pork loin prosciutto jerky pig boudin. Filet mignon pork chop frankfurter, pork loin kevin tri-tip porchetta spare ribs turducken andouille. Corned beef ground round andouille, pancetta boudin meatloaf ribeye meatball landjaeger cow filet mignon strip steak porchetta drumstick kevin. Swine ham landjaeger, venison fatback andouille bacon alcatra tail doner pork belly rump beef hamburger. Andouille shank landjaeger doner porchetta rump pancetta.
```
Comment (gmittal) [inline index.html:21]: I assume we're changing this?

File: css/index.css (hunk 2 above)
Comment (ankitr) [reply css/index.css]: whoops, missed that one. brb

File: js/main.js [hunk 4]
```diff
@@ -9,8 +9,7 @@ $('#more').click(function() {
     $('#info').animate({'opacity': '1'}, 190);
 });
 
-$('#back').click(function() {
-	$('section').css({'position': 'absolute'});
-    $('#info').animate({'opacity': '0.5'}, 200);
-    $('#info').animate({'left': '-110%'}, 200);
+$("#back").click(function(){
+	$("section").css({"position": "absolute"});
```
Comment (gmittal) [inline js/main.js]: Maybe some linting... haha (that extra indent is bothering me).

**Comment by gmittal** [comment]
Other than the main text, LGTM.

**Pull request merged**